platform/windows/display_server_windows.cpp

```diff
@@ -3117,6 +3123,12 @@ static BOOL CALLBACK _enum_proc_find_window_from_process_id_callback(HWND hWnd,
 			return TRUE;
 		}
 
+		wchar_t this_window_title[256];
+		GetWindowTextW(hWnd, this_window_title, sizeof(this_window_title) / sizeof(this_window_title[0]));
```

Comment:
Same here, instead of using a title which is not unique, it's probably better to use `SetWindowLongW()` and store the unique ID in extra window data (which can be the same ID used as `p_parent/p_embedding_id` in `create_window`).